<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="p_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="e_1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="e_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerScore">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemyScore">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enemy</string>
    </property>
   </widget>
   <widget class="QLabel" name="p_1_fon">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: silver;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="e_1_fon">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: silver;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="e_2_fon">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: silver;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="p_2_fon">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: silver;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>p_2_fon</zorder>
   <zorder>e_1_fon</zorder>
   <zorder>e_2_fon</zorder>
   <zorder>p_1_fon</zorder>
   <zorder>p_1</zorder>
   <zorder>e_1</zorder>
   <zorder>p_2</zorder>
   <zorder>e_2</zorder>
   <zorder>playerScore</zorder>
   <zorder>enemyScore</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
